releasenotes/notes/add-native-span-events-b04983442822d1b0.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Enables encoding span events as a top-level field in v0.4 payloads and introduces environment variable `DD_TRACE_NATIVE_SPAN_EVENTS`.
```

```suggestion
    Enables encoding span events as a top-level field in v0.4 payloads and introduces environment variable ``DD_TRACE_NATIVE_SPAN_EVENTS``.
```

changelog entries use rST not markdown